simple-backend/nlpviewer_backend/lib/stave_viewer.py

```diff
@@ -53,12 +54,23 @@ class StaveViewer:
     build path(deprecated in future update) and project path. By calling
     `run()`, StaveViewer starts a tornado server on a newly created thread.
     `open()` method will open the browser with the visualization.
-    
+
     Note: Currently the viewer mode is achieved by rerouting the django
     backend requests and it can be hard to maintain when structure changes
     in Stave. Refactoring is welcome to improve code logic and maintanence.
     """
 
+    ADMIN_USERNAME = "admin"
+    ADMIN_PASSWORD = "admin"
+    CONFIG_PATH = os.path.join(os.path.expanduser('~'), ".stave")
+    CONFIG_FILE = os.path.join(CONFIG_PATH, ".stave.conf")
```

Comment:
small comment: since we are using `.stave`, we don't need to add the dot before `.stave.conf`. The folder is already hidden.